src/cdk-experimental/menu/menu.ts

```diff
@@ -48,10 +56,42 @@ export class CdkMenu extends CdkMenuGroup implements AfterContentInit {
   @ContentChildren(CdkMenuGroup, {descendants: true})
   private readonly _nestedGroups: QueryList<CdkMenuGroup>;
 
+  /**
+   * A reference to the enclosing parent menu panel.
```

Comment:
Just to clarify, what's the difference between cdkMenu and cdkMenuPanel?

Reply:
cdkMenu is a directive attached directly to the menu content. cdkMenuPanel is attached to the ng-template. 

cdkMenuPanel provides a reference to the template used to display the content (cdkMenu) in an overlay. It also provides a reference to the child menu.